docs/plugins/discovery.asciidoc

```diff
@@ -26,7 +26,6 @@ The Google Compute Engine discovery plugin uses the GCE API for unicast discover
 
 A number of discovery plugins have been contributed by our community:
 
-* https://github.com/shikhar/eskka[eskka Discovery Plugin] (by Shikhar Bhushan)
```

Comment:
this plugin is unmaintained, and the only in this list that is not just adding hosts providers